packages/landing/static/examples/index.openlayers.wmts.3857.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>OpenLayers WMTS Basemaps Demo</title>
+    <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/css/ol.css"
+        type="text/css"
+    />
+    <script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.3.1/build/ol.js"></script>
+  </head>
+
+  <body>
+    <div id="map" style="width: 800px; height: 600px;"></div>
+    <script>
+     const apiKey = localStorage.getItem('api-key');
+     var attributions =
+         '<a href="https://www.linz.govt.nz/data/linz-data/linz-basemaps/data-attribution">LINZ CC BY 4.0 © Imagery Basemap contributors</a>';
+     const projstr = "EPSG:3857";
+     const startPos = [19467552, -5074414];
+     const startZoom = 6;
+     const wmtsUrl = "https://basemaps.linz.govt.nz/v1/tiles/aerial/EPSG:3857/WMTSCapabilities.xml?api="+apiKey;
+
+     //LINZ XYZ Services Map
+     const projection = ol.proj.get(projstr);
+     const projectionExtent = projection.getExtent();
+     const size = ol.extent.getWidth(projectionExtent) / 256;
+     const resolutions = new Array(14);
+     const matrixIds = new Array(14);
+     for (let z = 0; z < 14; ++z) {
+         resolutions[z] = size / Math.pow(2, z);
+         matrixIds[z] = z;
+     }
+
+     const parser = new ol.format.WMTSCapabilities();
+     const grid = new ol.tilegrid.WMTS({
+         origin: ol.extent.getTopLeft(projectionExtent),
+         resolutions: resolutions,
```

Comment:
We use const/short hands in other places we should be able to simplify these here too